lib/views/help/contact.html.erb

```diff
@@ -76,9 +76,27 @@
     <input class="houdini-input" type="radio" name="goals" id="goal3"
         <% if params["contact"] && params[:current_form] == 'writing-help' %>checked<% end %>>
     <div class="houdini-target contact-page__options">
+        <h3>If you need help writing a request, we've got you covered.</h3>
         <ul>
             <li>
-                Directly contact the team that run WhatDoTheyKnow:
+                Our <a href="<%= help_requesting_path %>">help pages</a> have 
+                guidance on how to make requests, including some top tips on 
+                how you can get the best response.
+            </li>
+            <li>
+                You can also find inspiration for your request on 
+                <a href="<%= link_to _("Read blog"), blog_path %>">our blog</a>.
+            </li>
+            <li>
+                <% if @user && @user.is_pro? %>
```

Comment:
-1 to this conditional; it's the same team, just different people _tend_ to respond based on the current user status. I don't want to have to differentiate this everywhere in the app. The contact form routes correspondence to the right inbox.

Reply:
I’d rather we didn’t differentiate at all, but we do elsewhere on the page.

I suppose you could take the view that a pro user might be less likely to use this form though… will post a fix up later.

Reply:
> I’d rather we didn’t differentiate at all, but we do elsewhere on the page.

Ah I didn't notice the existing conditional. Agree – would be happy for both to be removed!

Reply:
> Ah I didn't notice the existing conditional. Agree – would be happy for both to be removed!

Even better, less moving parts is always a maintainability bonus. 😃 

Fixup duly pushed in 479fabcb4eedb3d50f95ee6852bd0ab1fad3c198